Add Dice Seed Words templates

brandonblack opened the pull request.

brandonblack commented:
Some of the text in the README depends on #175 or similar being merged.

newtonick commented:
@brandonblack can you include the pip3 install steps or dependencies to use chisq.py? I haven't tried it yet, but looking at the imports on the top of the script I think it has a few outside dependencies.

brandonblack commented:
@newtonick just `scipy` -- I tried to keep it as close to simple python as reasonably practicable. Removed one unused import, and added a note on how to run it.

brandonblack commented:
My statistics are wrong, standby.

kdmukai commented:
A couple minor notes:

* I wouldn't call the PDFs "templates" as that implies something to be filled out. They're really more like look-up tables, reference tables, etc.

* The PDFs need some UI help. Once I stared at them a while I understood how they were meant to be used, but I'd find the process pretty mentally taxing. Not that it's difficult, exactly, it's more that for something really important you want it to be dead-simple to use so that it ideally adds zero mental load. Mental load == anxiety.

Maybe if more room was used to make the guide numbers more clear? I think it's fine if the PDFs end up taking up more pages. Like maybe prefix with the roll/step number:

```
        R2:1   R2:2   R2:3  . . .
R1:1
R1:2
R1:3
 .
 .
 .
```

Or labeling the roll steps with ABCD?
```
        B:1   B:2   B:3  . . .
A:1
A:2
A:3
 .
 .
 .
```
(I realize there may be complications when dealing with the different die possibilities)

Or in the worst case, just resort to an inefficient list of roll sequences:
```
4-1-5: foo
4-1-6: bar
4-2-1: meh
4-2-2: glorp
```

However they end up listed, I'd bias in favor of mental efficiency vs space/layout efficiency.

kdmukai commented:
True. I think there's probably no ideal UI solution, just a set of tough tradeoffs to try to balance just right.

I'm also starting to think that this is best presented in a separate repo/website/blog post since it's not directly integrated into anything on the SeedSigner side. The only update here would be in a README that points to your published resource as a better way to pick words (I will never cut out 2048 pieces of paper!).

If we eventually implemented your lookup tables within the SeedSigner UI, then obv that would be a different story (though it would probably make more sense to just implement the generalized dice roller UI we talked about at the conference: # of dice, min/max values of each one, we calc number of roll sets required).

brandonblack commented:
Yeah, I wasn't sure if you'd want these in the SeedSigner repo. I'll set something up.
